### ISSUE TITLE
Autoencoder only outputs greyscale

### ISSUE THREAD

**Issue by djdylex** [VainF/pytorch-msssim#26]
When i use SSIM with my autoencoder it produces greyscale images (or very close colors), yet mean square error gives me color. Is there a reason for this, i have tried taking SSIM per channel and i get the same issue. 

I am using the loss as:

ssim_loss = SSIM(data_range=1.0, size_average=True, channel=3)

Any suggestions?

**Comment by djdylex** [comment]
Nevermind, I was being an idiot. 

SSIM weights structure much more importantly than color, so when training first the structure will be learnt then the color. MSE does not do this.
